Trappist/src/Promact.Trappist.Test/Questions/QuestionsRepositoryTest.cs

```diff
@@ -219,7 +219,8 @@ public QuestionsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
                     CheckTimeComplexity = true,
                     RunBasicTestCase = true,
                     RunCornerTestCase = false,
-                    RunNecessaryTestCase = false
+                    RunNecessaryTestCase = false,
+                    LanguageList = new String[]{ "Java", "C" }
```

Comment:
Because I don't wanted to use all the coding language for testing. This is the reason why I got to know this bug.